sympy/series/limits.py

```diff
@@ -135,10 +135,11 @@ def __new__(cls, e, z, z0, dir="+"):
         if isinstance(dir, string_types):
             dir = Symbol(dir)
         elif not isinstance(dir, Symbol):
-            raise TypeError("direction must be of type basestring or Symbol, not %s" % type(dir))
-        if str(dir) not in ('+', '-'):
+            raise TypeError("direction must be of type basestring or "
+                    "Symbol, not %s" % type(dir))
+        if str(dir) not in ('+', '-', '+-'):
             raise ValueError(
-                "direction must be either '+' or '-', not %s" % dir)
+                "direction must be either '+' or '-' or '+-', not %s" % dir)
```

Comment:
"either" somewhat implies two items. I would write this as "direction must be one of '-', '+', or '-+'"